files/default/sql/ddl/3.3.0__initial_tables.sql

```diff
@@ -2295,21 +2295,32 @@ CREATE TABLE IF NOT EXISTS `statistics_comparison_config` (
     `strict` BOOLEAN DEFAULT FALSE,
     `relative` BOOLEAN DEFAULT FALSE,
     `threshold` FLOAT,
-    `compare_on` INT(11) NOT NULL,
+    `metric` INT(11) NOT NULL,
     PRIMARY KEY (`id`)
 ) ENGINE = ndbcluster DEFAULT CHARSET = latin1 COLLATE = latin1_general_cs;
 
+CREATE TABLE IF NOT EXISTS `job_schedule` (
+    `id` int NOT NULL AUTO_INCREMENT,
+    `job_id` int NOT NULL,
+    `start_datetime` timestamp,
+    `enabled` BOOLEAN NOT NULL,
+    `job_frequency` varchar(20) NOT NULL,
+    PRIMARY KEY (`id`),
+    UNIQUE KEY `job_id` (`job_id`),
+    CONSTRAINT `fk_schedule_job` FOREIGN KEY (`job_id`) REFERENCES `jobs` (`id`) ON DELETE CASCADE
+) ENGINE=ndbcluster DEFAULT CHARSET=latin1 COLLATE=latin1_general_cs;
+
 CREATE TABLE IF NOT EXISTS `feature_monitoring_config` (
     `id` INT(11) NOT NULL AUTO_INCREMENT,
     `feature_group_id` INT(11),
     `feature_view_id` INT(11),
-    `feature_name` VARCHAR(63) COLLATE latin1_general_cs NOT NULL,
+    `feature_name` VARCHAR(63) COLLATE latin1_general_cs DEFAULT NULL,
     `description` VARCHAR(2000) COLLATE latin1_general_cs DEFAULT NULL,
     `name` VARCHAR(255) COLLATE latin1_general_cs NOT NULL,
     `enabled` BOOLEAN DEFAULT TRUE,
     `feature_monitoring_type` tinyint(4) NOT NULL,
     `alert_config` VARCHAR(63) COLLATE latin1_general_cs, -- dummy this should become ref to another table
```

Comment:
now that we are not persisting alerts in this table for V1, should we keep this field for future use anyway or remove it ?